packages/driver/src/cypress/error_messages.coffee

```diff
@@ -615,15 +615,15 @@ module.exports = {
       Before the page load, you were bound to the origin policy:
         > #{originPolicy}
 
-      A cross origin error happens when your application navigates to a new superdomain which does not match the origin policy above.
+      A cross origin error happens when your application navigates to a new domain which does not match the origin policy above.
```

Comment:
```suggestion
      A cross origin error happens when your application navigates to a new URL which does not match the origin policy above.
```

since it's not necessarily the domain that's changed

Reply:
True story